src/python/bot/fuzzers/syzkaller/engine.py

```diff
@@ -0,0 +1,241 @@
+# Copyright 2020 Google LLC
```

Comment:
@oliverchang to review syzkaller directory and give comments.

Reply:
Review process with Oliver initiated.